lib/support/cli.js

```diff
@@ -231,6 +231,12 @@ module.exports.parseCommandLine = function parseCommandLine() {
       type: 'boolean',
       group: 'firefox'
     })
+    .option('firefox.windowRecorder', {
+      describe: 'Use the Firefox internal window recorder',
+      default: true,
```

Comment:
Really?

Reply:
For the mozilla fork, I think this should be enabled by default.  We have evidence that screen capture overhead can be quite significant and on Windows there isn't even any support for it at the moment.   Will probably need to flip it to false for an uplift to sitespeedio though.  Let me know what you think.